workspace.js

```diff
@@ -2,10 +2,10 @@ const GLib = imports.gi.GLib;
 const Clutter = imports.gi.Clutter;
 const Lang = imports.lang;
 const Main = imports.ui.main;
-const Mainloop = imports.mainloop;
 const Overview = imports.ui.overview;
-const Tweener = imports.ui.tweener;
 const Workspace = imports.ui.workspace;
+// const Mainloop = imports.mainloop;
```

Comment:
Can we just delete these if they're unused

Reply:
Sure, np.